addons/web/static/src/libs/bs5_utility_classes.scss

```diff
@@ -93,3 +93,21 @@ $position-properties: (
 .fst-normal {
   font-style: normal !important;
 }
+
+// == Line Height
+//------------------------------------------------------------------------------
+// "Bootstrap v5 ready" utility-class to handle line-height.
+// These class definition can be safely removed after migrating to BTS v5.
+
+$line-height-values: (
+  1: 1,
+  sm: $line-height-sm,
+  base: $line-height-base,
+  lg: $line-height-lg,
+);
+
+@each $-value-key, $-value in $line-height-values {
+  .lh-#{$-value-key} {
+    line-height: #{$-value};
+  }
+}
```

Comment:
@adr-odoo should I add all values or only the one used (and squash those commits)?

Reply:
Putting all values is less confusing I think.

Reply:
Or maybe only the one used (in this case `.lh-base`) ?

Reply:
Putting all values is less confusing I think.